airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/MetricClientFactory.java

```diff
@@ -67,6 +71,45 @@ public static synchronized void initialize(final MetricEmittingApp metricEmittin
     }
   }
 
+  private static StatsdConfig getDatadogStatsDConfig() {
+    return new StatsdConfig() {
+
+      /**
+       * @return
+       */
+      @Override
+      public String host() {
+        return configs.getDDAgentHost();
+      }
+
+      /**
+       * @param key Key to lookup in the config.
+       * @return
+       */
+      @Override
+      public String get(String key) {
```

Comment:
why is this returning null?

Reply:
Using the default settings. 

Doesn't mean anything - unless we want to inject customized configs in it, but in this case temporal side wouldn't consume anything anyway.

Reply:
Gotcha. Can we clean up the javadocs here to make it clear what the methods are used for? e.g. make it clear the `get` method is supposed to be stubbed out, and the `host` method is hardcoded.